### ISSUE TITLE
Arduino nano 33 BLE not visible to clients in bluetooth scan when powered up with 3V3 (needs 5V initialization) 

### ISSUE THREAD

**Issue by reniar1** [arduino-libraries/ArduinoBLE#301]
Hey guys,

I am using the Arduino nano 33 BLE and I am powering it up with 3.3 volt and I cut the jumper to make that possible.

However if I start the board (peripheral) with 3v3 it will not be visible to the clients when they scan for Bluetooth.

However if I afterwards plugin the 5v cable for a few seconds (micro USB) it will be visible, I can then unplug it again and it (the peripheral) will still be visible to the clients when they scan for bluetooth.

It feels/seems as if the Arduino nano 33 BLE needs the 5volt initially (provided in example through the USB port) to start up the Bluetooth service and then it can be constantly powered by 3v3. instead of 5volt.

In summary if you start up the peripheral nano 33 BLE board with 3v3 it cannot be found by clients in Bluetooth.

Do you have any knowledge about this behavior?

Ideally I would like to power the board with 3v3 and then see it (as an peripheral device) on my clients without having to initialize it by plugin in the 5v cable one time.
